specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01-preview/botservice.json

```diff
@@ -2137,6 +2141,10 @@
           "type": "string",
           "description": "The Slack verification token. Value only returned through POST to the action Channel List API, otherwise empty."
         },
+        "scopes": {
```

Comment:
>scopes [](http://example.com/codeflow?start=9&length=6)

How are multiple values separated? Generally collections should not be encoded in strings if their members need to be consumed indvidually.

Reply:
@youyu16 can you respond to this?

Reply:
it will separated by ",", example, "channels:history,channels:read"

Reply:
OK, I'd recommend updating the description to explain the format, but won't block ARM signoff.